src/09-reveal/style.scss

```diff
@@ -0,0 +1,17 @@
+@charset "UTF-8";
+/*********************************************************************************************
+
+Name: Reveal
+Author:  Ministry of Justice
+
+**********************************************************************************************/
+
+/**
+* This file contains imported styling to support GOV UK Frontend usage in the plugin
+*
+* To import styles from govuk-frontend, simply include the Gov UK module using
+*  @imports with a URL reference
+**/
+
+// https://design-system.service.gov.uk/components/accordion/
```

Comment:
Might want to swap this for detail component?

Reply:
Done